.github/workflows-source/ci-main.yml

```diff
@@ -199,6 +182,36 @@ jobs:
         run: |
           sudo rm -rf /private/var/tmp/bazel-output
 
+  bazel-test-macos-namespace:
```

Comment:
It looks like namespace is our de facto platform for running Darwin jobs now, this could just be `bazel-test-macos`

Reply:
How about just naming the CI jobs in way that's clear whether it's arm64 or x86-64?